sample.env

```diff
@@ -22,3 +22,5 @@ COOKIE_SECRET='this is a secret'
 
 PEER=stuff
 DEBUG=true
+
+IMAGE_BASE_URL='https://s3.amazonaws.com/freecodecamp/'
```

Comment:
Is this supposed to be image specific path instead of the parent directory?

Reply:
It is specific to images. I used the path Quincy gave me in the convo for issue #15941. But if there is a sub-directory always used for images, it would probably be better to include that in the string too.

Reply:
In that case, I would recommend confirming on main issue thread instead. Also we can (rather should) update `*.env` or any documentations only after a PR has landed. Its cleaner, and ensures that we do not miss anything that is actually implemented.

I'll keep this PR blocked, and let you proceed with the work, and then we can revisit documenting and working on the chores? What do you say?